hr_family/__manifest__.py

```diff
@@ -21,11 +21,11 @@
 
```

Comment:
@GillesTephaneMeyomesse Thanks for the contribution, can you update with Small Headers..